src/frontend/efiling-frontend/src/components/page/package-review/PackageReview.js

```diff
@@ -1,30 +1,142 @@
-import React, { useEffect } from "react";
+import React, { useEffect, useState } from "react";
+import moment from "moment";
 import PropTypes from "prop-types";
-import { Header, Footer, Button } from "shared-components";
+import { Header, Footer, Button, Table, Alert } from "shared-components";
+import { MdCancel } from "react-icons/md";
 import axios from "axios";
 import { propTypes } from "../../../types/propTypes";
 
 export default function PackageReview({ page: { header, packageId } }) {
+  const [error, setError] = useState(false);
+  const [packageNoDetails, setPackageNoDetails] = useState([
+    {
+      name: "Package Number:",
+      value: "",
+      isNameBold: false,
+      isValueBold: true,
+    },
+    { name: "Submitted By:", value: "", isNameBold: false, isValueBold: true },
+    {
+      name: "Submitted Date:",
+      value: "",
+      isNameBold: false,
+      isValueBold: true,
+    },
+  ]);
+  const [courtFileDetails, setCourtFileDetails] = useState([
```

Comment:
Do we need to initialize the array here since we set the variable on successful api call? Would `useState([])` be equivalent?

Reply:
Great question.  The way the shared-component table works, is that it takes in a list of key/value pairs and converts them to a table.  If we don't initialize the array and if there is an error parsing the values or the api is down or whatever, then the table won't show up at all.  We need at least the keys in the array to display the table at all.

Reply:
yeah makes sense. I think if the api does not return any response we will redirect the user back to the parent application, but as we don't have this logic yet we should leave it as is for now, agree with you